vllm/worker/model_runner.py

```diff
@@ -1060,7 +1062,8 @@ def capture(
         # NOTE(woosuk): Python 3.8 does not support multi-line with statements.
         # https://stackoverflow.com/questions/31039022/python-multi-line-with-statement
         self._graph = torch.cuda.CUDAGraph()
-        with torch.cuda.graph(self._graph, pool=memory_pool):  # noqa: SIM117
+        with torch.cuda.graph(self._graph, pool=memory_pool,
+                              stream=stream):  # noqa: SIM117
```

Comment:
QQ: What is `noqa: SIM117` for?

Reply:
Maybe it's because of my comment above, which is not relevant any more?

Reply:
It is not introduced in this PR. fixed in [b1767f2](https://github.com/vllm-project/vllm/pull/4818/commits/b1767f292cc73e2f8671138d85e9b432d592365b)